nixos/tests/go-neb.nix

```diff
@@ -34,10 +35,9 @@ import ./make-test-python.nix ({ pkgs, ... }:
     start_all()
     server.wait_for_unit("go-neb.service")
     server.wait_until_succeeds(
-        "curl -fL http://localhost:4050/services/hooks/d2lraXBlZGlhX3NlcnZpY2U"
-    )
-    server.wait_until_succeeds(
-        "journalctl -eu go-neb -o cat | grep -q service_id=wikipedia_service"
+        "curl -fL http://localhost:4050/services/hooks/d2lraXBlZGlhX3NlcnZpY2U",
+        "journalctl -eu go-neb -o cat | grep -q service_id=wikipedia_service",
+        "grep -q changeme /var/run/go-neb/config.toml",
```

Comment:
TypeError: wait_until_succeeds() takes 2 positional arguments but 4 were given

Reply:
Not sure what happened. I did so many things at the same time, that I forgot to re-test it again.
Now it works on my machine.